### PR TITLE
WMCO 10.15.0 release notes

### PR DESCRIPTION
**Pull request by mburke5678**
Errata: https://errata.devel.redhat.com/docs/show/120235

Preview: https://71436--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-10-15-x#wmco-10-15-0

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mburke5678** [comment]
@rrasouli @asahay19 Can you PTAL at the new features and bug texts in this PR.

File: windows_containers/windows-containers-release-notes-10-15-x [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+// Also in 7.2.1
+* Previously, the WMCO was missing a field required for draining nodes with emptydir volumes attached. As a consequence, customers that had nodes with emptydir data volumes would see the following error in the logs: `cannot delete Pods with local storage`. The necessary field was added to the node drain helper struct in the WMCO. As a result, customers are able to drain nodes with emptydir volumes attached. (link:https://issues.redhat.com/browse/OCPBUGS-27300[*OCPBUGS-27300*])
+
+* Because of a lack of synchronization between Windows machine set nodes and BYOH instances, during an upgrade the machine set nodes and the BYOH instances could upgrade simultaneously, which could impact running workloads. This fix introduces a locking mechanism so that machine set nodes and BYOH instances upgrade individually. (link:https://issues.redhat.com/browse/OCPBUGS-8996[*OCPBUGS-8996*])
```
Comment (rrasouli) [inline windows_containers/windows-containers-release-notes-10-15-x:72]: machineset nodes and BYOH instances

File: windows_containers/windows-containers-release-notes-10-15-x [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+// Also in 7.2.1,
+* Previously, the WMCO did not properly wait for Windows VMs to finish rebooting. This led to occasional timing issues where the WMCO would attempt to interact with a node that was in the middle of a reboot, causing WMCO to log an error and restart node configuration. Now, the WMCO waits for the instance to completely reboot. (link:https://issues.redhat.com/browse/OCPBUGS-17217[*OCPBUGS-17217*])
+
+// Also in 7.2.1
+* Previously, the WMCO was missing a field required for draining nodes with emptydir volumes attached. As a consequence, customers that had nodes with emptydir data volumes would see the following error in the logs: `cannot delete Pods with local storage`. The necessary field was added to the node drain helper struct in the WMCO. As a result, customers are able to drain nodes with emptydir volumes attached. (link:https://issues.redhat.com/browse/OCPBUGS-27300[*OCPBUGS-27300*])
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-10-15-x:70]: @rrasouli Would that be the `DeleteEmptyDirData: true` field?

**Review by abrennan89 — COMMENTED** [review]
Some comments and suggestions. Great work overall!

File: windows_containers/windows-containers-release-notes-10-15-x.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+= {productwinc} release notes
+include::_attributes/common-attributes.adoc[]
+:context: windows-containers-release-notes
+
+toc::[]
+
+[id="about-windows-containers"]
+== About {productwinc}
+
+Windows Container Support for Red{nbsp}HatHat OpenShift enables running Windows compute nodes in an {product-title} cluster. Running Windows workloads is possible by using the Red{nbsp}HatHat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-10-15-x.adoc:12]: @abrennan89 Good catch on the HatHat! The official name of the feature is Windows Container Support for Red Hat OpenShift. I was just putting in the `{nbsp}` that I think we are supposed to be using.

File: windows_containers/windows-containers-release-notes-10-15-x.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.0 were released in {UPDATE WHEN AVAILABLE} link:https://access.redhat.com/errata/RHSA-2023:4025[RHSA-2023:4025] {UPDATE WHEN AVAILABLE}
+
+=== New features and improvements
+[id="wmco-10-15-0-node-certificates"]
+
+[id="wmco-10-15-0-numbering"]
+==== New WMCO numbering
+
+Starting with this release, y-stream releases of the WMCO will be in step with {product-title}, with only z-stream releases between {product-title} releases. The WMCO numbering will reflect the associated {product-title} version in the y-stream position. For example, the current release of WMCO is associated with {product-title} version 4.15. Thus, the numbering is WMCO 10.15.z.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-10-15-x.adoc:42]: @abrennan89 I'm thinking I should leave this. The wording was approved by the WMCO team.

File: _topic_maps/_topic_map.yml [hunk 5]
```diff
@@ -2522,7 +2522,7 @@ Topics:
 - Name: Red Hat OpenShift support for Windows Containers overview
   File: index
 - Name: Red Hat OpenShift support for Windows Containers release notes
-  File: windows-containers-release-notes-8-x
+  File: windows-containers-release-notes-10-15-x
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply _topic_maps/_topic_map.yml:2525]: @abrennan89 No. The `File: windows-containers-release-notes-8-x` is in main only. This is a new release so we aren't really replacing anything in the live docs. Good question though!

**Pull request merged**